src/tests/ftest/util/launch_utils.py

```diff
@@ -1233,6 +1294,12 @@ def run_tests(self, logger, result, repeat, slurm_setup, sparse, fail_fast, stop
                 # Stop logging to the test log file
                 logger.removeHandler(test_file_handler)
 
+        # Cleanup any specified files at the end of testing
+        for file, info in cleanup_files.items():
+            command = command_as_user(f"rm -fr {file}", info['user'])
+            if not run_remote(logger, info['hosts'], command).passed:
+                return_code |= 16
```

Comment:
Also, at least on Aurora we sometimes run the harness setup test to get configs, etc. in place. This now means we can't do that

Reply:
The configs would still accessible - in `/var/tmp/daos_testing/configs` by default - it would just mean a normal `systemctl start daos_server` would not find this config unless a `sudo cp /var/tmp/daos_testing/configs/daos_server.yml /etc/daos/` was issued beforehand.

Details aside, we could add yet another launch.py option to control bypass this cleanup step.

Reply:
Would it make sense to archive the systemctl config file if launch.py is run w/ `-a`?

Reply:
Let me think about this more... I'm on the fence